shell2http.go

```diff
@@ -294,8 +294,11 @@ func getShellHandler(appConfig Config, path string, shell string, params []strin
 			shellOut, err = osExecCommand.Output()
 		}
 
-		if err != nil && !appConfig.showErrors {
+		if err != nil {
 			log.Println("exec error: ", err)
+		}
+
+		if err != nil && !appConfig.showErrors {
```

Comment:
> But it seems it will be a complication. I wanted to keep it simple.
 
Agreed - I definitely didn't want to suggest complicating things ;-). The main thought was allowing users calling the API programmatically to be able to figure out whether the command was actually successful or not.

But obviously that should be included in the `<html>` block, not before it, so my code suggestion does not make sense ;-)

Another thought might be to indicate success or failure of the command via a response header and/or response code?